cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,491 @@ Cask Data Application Platform Release Notes
+  XMLMultiParser plugins.
+
+- :cask-issue:`HYDRATOR-1177` - Plugins can now retrieve the input and output schema of
+  their stage in their initialize methods.
+
+- :cask-issue:`HYDRATOR-1323` - Added a new ErrorTransform plugin-type that can be placed
+  after a pipeline stage to consume errors emitted by that stage.
+
+- :cask-issue:`WRANGLER-3` - The CDAP UI's Wrangler modal dialog will give a warning when
+  you try to close or exit out of it without confirmation.
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-2543` - Fixed an issue of a hanging application in the case that a
+  user program JAR is missing dependencies.
+
+- :cask-issue:`CDAP-4739` - Fixed an issue to make artifact, datasets, logs, and
+  coprocessor jar locations resilient to an HA upgrade.
+
+- :cask-issue:`CDAP-5107` - Added an HTTP RESTful endpoint to retrieve a specific property
+  for a specific version of an artifact in the 'system' scope.
+
+- :cask-issue:`CDAP-5717` - Fixed an issue with starting the CDAP CLI and the CDAP
+  Standalone when the on-disk path has a space in it.
+
+- :cask-issue:`CDAP-6642` - Attempting to delete a system artifact by specifying a user
+  namespace (that previously returned a 200, even though the artifact was not deleted) will
+  now return a 404, as that combination of system and user will never occur.
+
+- :cask-issue:`CDAP-6690` - Improved the formatting of dataset instance properties in the
+  output of the CDAP CLI.
+
+- :cask-issue:`CDAP-6704` - Improved and clarified certain of the CDAP CLI help text and
+  its error messages.
+
+- :cask-issue:`CDAP-7155` - Fixed a problem where the Dataset Service failed to start up
+  if authorization was enabled and the authorization plugin was slow to respond.
+
+- :cask-issue:`CDAP-7228` - Empty and null metadata tags are now removed in the metadata
+  upgrade step of the CDAP Upgrade Tool.
+
+- :cask-issue:`CDAP-7302` - Fixed an issue that caused the CDAP Master to die if HBase was
+  down when a follower became the leader.
+
+- :cask-issue:`CDAP-7694` - Fixed an issue where the CDAP service scripts could cause a
+  terminal session to not echo characters.
+
+- :cask-issue:`CDAP-7813` - The security policies for accessing entities have been changed
+  and the documentation updated to reflect these changes.
+
+- :cask-issue:`CDAP-7911` - The error messages returned for bad requests to the metadata
+  search RESTful APIs have been improved.
+
+- :cask-issue:`CDAP-7930` - Performing a metadata search now returns the correct total,
+  even if the offset is very large.
+
+- :cask-issue:`CDAP-7935` - Fixed an issue with the CDAP Standalone not starting and
+  stopping correctly.
+
+- :cask-issue:`CDAP-7991` - The Cask Market now shows only those entities that are valid
+  for the specific version of CDAP viewing them.
+
+- :cask-issue:`CDAP-8001` - Fixed an issue with logs when a namespace was deleted and then
```

Comment:
Yes, but after reading the issue, I'm not certain what the symptoms are. I _think_ what it is saying is:
Deleting a namespace does not delete its log meta data, resulting in old log data appearing when the namespace is recreated with the same name. Is that it?